lxd/events/common.go

```diff
@@ -45,7 +45,7 @@ func (e *listenerCommon) heartbeat() {
 
 	defer e.Close()
 
-	pingInterval := time.Second * 5
+	pingInterval := time.Second * 10
```

Comment:
Out of interest why was this needed?

Reply:
I've seen quite a few cases now where we seem to be hitting the timeout during initial forwarding through a busy LXD cluster.

Reply:
interesting, thanks